install_ffmpeg.sh

```diff
@@ -25,6 +25,25 @@ if [ ! -e "$HOME/x264/x264" ]; then
   make install-lib-static
 fi
 
```

Comment:
here need to add the script to install nv-codec-headers.

Reply:
yes, exactly. It must be added. Thanks, @oscar-davids